zest/releaser/pypi.py

```diff
@@ -304,6 +304,62 @@ def extra_message(self):
             return default
         return result
 
+    def history_file(self):
```

Comment:
In combination with `self._get_text()` I originally thought "we're grabbing the textual content of the history file".
The cause for the misunderstanding is that I'm only seeing the changes and not the whole :-)

But... perhaps `history_file_path` might be a tad clearer. Depends on the other method names, of course. I should read the whole source code again some day as I don't know it from the top of my head :-)

Reply:
The code was already looking in the config for `history_file`, so I say it is best that we keep it that way.